test/integration_test/applicationbackup_test.go

```diff
@@ -859,6 +863,23 @@ func applicationBackupSyncControllerTest(t *testing.T) {
 	destroyAndWait(t, []*scheduler.Context{appCtx})
 }
 
+func waitForAppBackupCompletion(name, namespace string) error {
+	getAppBackup := func() (interface{}, bool, error) {
+		appBackup, err := k8s.Instance().GetApplicationBackup(name, namespace)
+		if err != nil {
+			return "", false, err
+		}
+
+		if appBackup.Status.Status == storkv1.ApplicationBackupStatusSuccessful {
```

Comment:
Can stop retrying if it goes to Failed state

Reply:
I got the condition wrong there it should be 
`if appBackup.Status.Status != storkv1.ApplicationBackupStatusSuccessful` {
.............
.............
}`